tests/general/InvalidLinkBearTest.py

```diff
@@ -10,18 +10,23 @@
 
 def custom_matcher(request):
```

Comment:
could you state in the commit message that you are documenting this function exactly? IMO its a little vague right now.